flags_test.go

```diff
@@ -387,6 +394,47 @@ func TestHandleDeactivateCommandWithForce(t *testing.T) {
 	assert.Equal(t, expected, flags.Command)
 }
 
+func TestHandleLocalDeactivationWithControlModeNotOne(t *testing.T) {
+	args := []string{"./rpc", "deactivate", "-local"}
+	flags := NewFlags(args)
+	flags.amtCommand.PTHI = MockPTHICommands{}
```

Comment:
i dont see where mode and result are set here like the others -- its good to be explicit to avoid side-effects instead of relying on what you think the default values are.

Reply:
done